src/celeritas/em/distribution/WentzelDistribution.hh

```diff
@@ -156,13 +156,11 @@ CELER_FUNCTION Real3 WentzelDistribution::operator()(Engine& rng) const
 
         if (!BernoulliDistribution(g_rej)(rng))
         {
-            return {0, 0, 1};
+            return 0;
```

Comment:
Wait, I thought this should be a *forward* scatter (no interaction) versus a 90 degree scatter?

Maybe instead:
```suggestion
            // Reject scattering: particle does not change direction
            cos_theta = 1;
```

Reply:
Yes you're completely right, I accidentally was thinking about returning the angle as 0 degrees